games/BRICK DODGER.js

```diff
@@ -0,0 +1,261 @@
+  [brickBreak, bitmap`
+................
+................
+................
+................
+................
+................
+................
+................
+................
+................
+................
+................
+........000.....
+......00CCC0....
+.00.00CCCCC0.00.
+0CC0CCCCCCCC0CC0`],
+  [bg, bitmap`
+2222222222222222
+1111121111111211
+1111121111111211
+1111121111111211
+2222222222222222
+1211111112111111
+1211111112111111
+1211111112111111
+2222222222222222
+1111121111111211
+1111121111111211
+1111121111111211
+2222222222222222
+1211111112111111
+1211111112111111
+1211111112111111`],
+  [bg2,bitmap`
+1111111111111111
+1111111111111111
+1111111111111111
+1111111111111111
+1111111111111111
+1111111111111111
+1111111111111111
+1111111111111111
+1111111111111111
+1111111111111111
+1111111111111111
+1111111111111111
+1111111111111111
+1111111111111111
+1111111111111111
+1111111111111111`],
+);
+const level = [
+  map`
+....
+....
+....
+....`,
+  map`
+.......
+.......
+.......
+.......
+.......
+...d...`
+];
+function reset(){
+  score = 1, isGameOver = false, delay = 150, plrX = 2, anim = false, maxY = 3, isPlaying = false;
+  clearText();
+  setMap(level[0]);
+  setBackground(bg);
+  addSprite(0,0,brick);
+  addSprite(plrX,maxY,plr);
+  playerSprite = getFirst(plr);
+  brickSprite = getFirst(brick);
+}
+
+onInput("a", () => {
+  if (plrX != 0 && isGameOver == false){
+    plrX -= 1;
+    const dodgePlayback = playTune(dodge)
+  }
+});
+onInput("d", () => {
+  if (plrX != 3 && isGameOver == false){
+    plrX += 1;
+    const dodgePlayback = playTune(dodge)
+  }
+});
+onInput("k", () => {
+  if (isGameOver == true){
+    isGameOver = false; 
+    reset(); 
+    loop();
+  }
+});
+
+function loop(){
+  if (isGameOver==false){
+    playerSprite.x = plrX;
+    playerSprite.y = maxY;
+    brickSprite = getFirst(brick);
+      
+    if (brickSprite.y == height()-1){
+      if(Math.floor(Math.random() * 3) == 2){
+        addSprite(Math.floor(Math.random() * width()),0,brick);
+      }else{
+        addSprite(plrX,0,brick);
+      }
+      score++;
+      if (delay > 40){
+        delay - 1
+      }
+      addSprite(brickSprite.x,brickSprite.y,brickBreak);
+      brickSprite.remove()
+      //const brickBreakPlayback = playTune(breakMusic)
+    }else{
+      brickSprite.y += 1
+      if (brickSprite.y == height()-2 && score > 1){
+        getFirst(brickBreak).remove()
+      }
+    }
+    if (!isGameOver) {
+      setTimeout(loop, delay);
+    }
+    if(plrX == brickSprite.x){
+      if(brickSprite.y == maxY){
+        gameOver();
+      }
+    }
+  }
+}
+function gameOver(){
+  //bgMusic.end()
+  playerSprite.remove()
+  setBackground(bg2);
+  const endPlayBack = playTune(endMusic);
+  isGameOver = true; 
+  setMap(level[1]);
+  addText(`BRICK DODGER`, { x: 2, y: 2, color: color`0` });
+  addText(`Score:${score}`, { x: 2, y: 4, color: color`0` });
+  addText(`Thx for playing`, { x: 2, y: 7, color: color`6` });
```

Comment:
just one thing, I think it's better to avoid abbreviations, to fit the screen, adjust Y, being like this:
`addText(`Thanks for playing`, { x: 1, y: 7, color: color`6` });`

Reply:
Yeah that looks much better. Do you need me to change this, or is it already changed( I'm new to github )?

Reply:
yes, I need you to change

`Files changed` > `...` > `edit`